FXyz-Importers/src/main/java/org/fxyz3d/importers/Importer3D.java

```diff
@@ -90,13 +87,18 @@ public static Node load(String fileUrl, boolean asPolygonMesh) throws IOExceptio
      * @return The loaded Node which could be a MeshView or a Group and the Timeline animation
      * @throws IOException if issue loading file
      */
-    public static Pair<Node,Timeline> loadIncludingAnimation(String fileUrl, boolean asPolygonMesh) throws IOException {
+    public static Model loadIncludingAnimation(URL fileUrl, boolean asPolygonMesh) throws IOException {
```

Comment:
Having a `load`/`loadAsPublic method that will call `loadIncludingAnimation`, I'd make the latter private?

Reply:
agreed